arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -87,15 +131,15 @@
             "in": "body",

```

Comment:
Please add descriptions
put: ""Asynchronously creates a new container registry with the specified parameters."
resourceGroup: "The name of the resource group within the user's subscription."
registryName: "The name of the container registry."
registryCreateParameters: "The parameters to provide for the created container registry."